cytominer_database/ingest.py

```diff
@@ -153,13 +157,8 @@ def seed(source, target, config_path, skip_image_prefix=True):
 
         # ingest the image CSV
         try:
-            into(
-                input=image,
-                output=target,
-                name=name.capitalize(),
-                identifier=identifier,
-                skip_table_prefix=skip_image_prefix,
-            )
+            into(input=image, output=target, name=name.capitalize(), identifier=identifier, con=con,
```

Comment:
this prob needs some linting?

Reply:
Thanks, the formatting changed and we gained a parameter here. I'll try to keep the existing formatting in another change.